web_scraping/README.md

```diff
@@ -2,4 +2,14 @@
 
 The code in this example demonstrates how Beam functions can be used to scrape Wikipedia pages in parallel.
 
-Run this example: `python app.py`
+## Mapped Crawl
```

Comment:
title should be in layman's terms

Reply:
instead of explaining what it is internally, explain it from a user's perspective i.e. "running a parallelized web scraper"

as a thought experiment, you can look at anything you write and ask, "would a user explain it like this?"